Patches/ClientOptionsPatch.cs

```diff
@@ -23,8 +23,9 @@ public static class OptionsMenuBehaviourStartPatch
     private static ClientOptionItem SwitchVanilla;
 #if DEBUG
     private static ClientOptionItem VersionCheat;
+    private static ClientOptionItem GodMode;
+    private static ClientOptionItem AutoRehost;
 #endif
```

Comment:
If dev versions are built as debug, then these settings will appear for users of the dev version

Reply:
I dont think we are building debug...